docs/lib/datastore/fragments/flutter/relational/save-many-snippet.md

```diff
@@ -1,11 +1,14 @@
 ```dart
-TBD
+Post post = Post(title: "My First Post");
+User editor = User(username: "Nadia");
+PostEditor postEditor = PostEditor(post: post, editor: editor);
+
+try {
+  await Amplify.DataStore.save(post);
+  await Amplify.DataStore.save(editor);
+  await Amplify.DataStore.save(postEditor);
```

Comment:
Maybe something like:

```dart
await Stream.fromIterable([post, editor, postEditor])
    .forEach((model) => await Amplify.DataStore.save(model))
```

Reply:
Kept similar to the [JS version](https://docs.amplify.aws/lib/datastore/relational/q/platform/js#many-to-many) of the docs. Added step wise comments of saving first post, then editor and finally postEditor. Here the order matters and I think it's explicit this way.